web/src/main/java/com/microsoft/applicationinsights/web/internal/WebRequestTrackingFilter.java

```diff
@@ -204,7 +202,6 @@ private void configureWebAppNameContextInitializer(String appName, TelemetryConf
      */
     public void destroy() {
         cleanup();
-        ThreadContext.remove();
```

Comment:
What did it use to do?

Reply:
it's redundant. It's called in `cleanup()`